lib/rubocop/packs.rb

```diff
@@ -23,47 +23,63 @@ class Error < StandardError; end
 
     private_constant(:CONFIG_DEFAULT, :PROJECT_ROOT)
 
+    class Offense < T::Struct
+      extend T::Sig
+
+      const :cop_name, String
+      const :filepath, String
+
+      sig { returns(ParsePackwerk::Package) }
+      def pack
+        ParsePackwerk.package_from_path(filepath)
+      end
+    end
+
+    sig { params(paths: T::Array[String], cop_names: T::Array[String]).returns(T::Array[Offense]) }
+    def self.offenses_for(paths:, cop_names:)
```

Comment:
The idea here is that when we move files, we will call `offenses_for` on the moved files after moving over the known rubocop errors. Then, we can take the offenses and add them to the `package_rubocop_todo.yml` file of the destination pack.